### PR TITLE
docs(faq): add missing PS docs link

### PR DESCRIPTION
**Pull request by icy-comet**
### Prerequisites

- [x] I have read and understood the `CONTRIBUTING` guide
- [x] The commit message follows the [conventional commits][cc] guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added / updated (for bug fixes/features)

### Description
Added missing link to the PowerShell docs for terminal support reference.

### CODE REVIEW

File: docs/docs/faq.mdx [hunk 1]
```diff
@@ -183,11 +183,11 @@ export VIRTUAL_ENV_DISABLE_PROMPT=1
 ### Text decoration styles like bold and dimmed don't work
 The text decoration styles are based on your terminal's capabilities.
 
-A quick way to check if your terminal supports the specific style escape sequence is to take a look at the `terminfo` database, on Linux.
+A quick way to check if your terminal supports a specific style escape sequence is to take a look at the `terminfo` database, on Linux.
 Refer [this page on ArchWiki](https://wiki.archlinux.org/title/Bash/Prompt_customization#Terminfo_escape_sequences).
 
 If you are on Windows, use Windows Terminal. It closely copies the `xterm-256color` capabilities.
-See the [PowerShell docs] and [this GitHub comment](https://github.com/microsoft/terminal/issues/6045#issuecomment-631743728).
+See the [PowerShell docs](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_ansi_terminals?view=powershell-7.2) and [this GitHub comment](https://github.com/microsoft/terminal/issues/6045#issuecomment-631743728).
```
Comment (JanDeDobbeleer) [inline docs/docs/faq.mdx]: Add the links below like the others:

```markdown
[PowerShell docs][ps-docs] and and [this GitHub comment][gh-comment]

...
[ps-docs]: https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_ansi_terminals?view=powershell-7.2
[gh-comment]: https://github.com/microsoft/terminal/issues/6045#issuecomment-631743728
```

Comment (JanDeDobbeleer) [reply docs/docs/faq.mdx]: Same for arch above, missed that initially.

**Review by JanDeDobbeleer — CHANGES_REQUESTED** [review]
Just a remark on the link style. We always use bottom links.

File: docs/docs/faq.mdx (hunk 1 above)
Comment (icy-comet) [reply docs/docs/faq.mdx]: Pushed a new commit. Sorry for the inconsistent styling.

**Review by JanDeDobbeleer — APPROVED** [review]

**Pull request merged**